xml/http_wwwauth.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!-- HTTP WWW-Authenticate headers are matched against these patterns to fingerprint HTTP servers. -->
-<fingerprints matches="http_header.wwwauth">
+<fingerprints matches="http_header.wwwauth" protocol="http" database_type="service" preference="0.20">
```

Comment:
I'm asking this both because I don't agree with the preference value and I think it might be useful to understand how you chose this value, since the introduction of this value is a key part of this PR.  The regular expressions used in this file seem to be fairly restrictive -- they have beginning and end anchors and look for very specific strings that are unlikely to match when applied to arbitrary other things.